lib/xdparser2.ts

```diff
@@ -674,94 +674,130 @@ function parseSplitsFromAnswer(answerWithSplits: string, splitCharacter?: string
   return splits
 }
 
-export function inlineMarkdownParser(str: string): MDClueComponent[] {
-  const components: MDClueComponent[] = []
-  let token = ""
-  let mode: MDClueComponent[0] = "text"
-  let linkText = ""
+function isAlphabetical(str: string): boolean {
+  return str.charCodeAt(0) >= 97 && str.charCodeAt(0) <= 122
```

Comment:
This would be clearer if you just used a regex (/^[a-z]/.test(str)). But should this actually be more permissive than just checking a-z? What about diacritics, etc.

Reply:
eh i think this is fine. i can always change if we need to be more permissive